pkg/operator/controller/ingress/controller.go

```diff
@@ -1277,7 +1284,34 @@ func computeUpdatedInfraFromService(service *corev1.Service, infraConfig *config
 	if platformStatus == nil {
 		return false, fmt.Errorf("invalid PlatformStatus within Infrastructure config")
 	}
+	ingresses := service.Status.LoadBalancer.Ingress
+	ingressLBIPs := []configv1.IP{}
+	for _, ingress := range ingresses {
+		if len(ingress.IP) > 0 {
```

Comment:
I don't think this will work as AWS doesn't set `status.loadBalancer.ingress.ip`, it only sets `status.loadBalancer.ingress.hostname`. AWS doesn't use IPs for their load balancer, only hostnames. GCP uses IP addresses.

Are you aware of this? The API design doesn't seem to accommodate hostnames for AWS https://github.com/openshift/api/pull/2046. Am I mising something?

Reply:
Thanks for pointing this out. I have updated my fix to account for the fact that only the `Hostname` of the Ingress LB is available on AWS.